core/base/src/main/java/alluxio/security/authorization/AccessControlList.java

```diff
@@ -86,6 +86,9 @@ public AccessControlList() {
 
   /**
    * Clears out all entries (does not modify the owner name and owning group).
+   * <p>
+   * Sets the ACL {@link #mMode} to {@code 0} and the {@link #mExtendedEntries}
```

Comment:
Unnecessary links to `mMode` and `mExtendedEntries`. Try to explain with words the result of the assignments of these variables without referencing implementation. What clearing the entries represent? No access? Full access? Try to find usages to figure this out.